tests/test_utils.py

```diff
@@ -152,3 +152,21 @@ def models_are_equivalent(model_a: TopLevelOscalModel, model_b: TopLevelOscalMod
     # this will change the second model as a side-effect
     model_b.metadata.last_modified = model_a.metadata.last_modified
     return model_a == model_b
+
+
+def text_files_equal(path_a: pathlib.Path, path_b: pathlib.Path) -> bool:
+    """Determine if files are equal, ignoring newline style."""
```

Comment:
I was surprised I had put this in utils.  I had meant to put it in test_utils I think.  Anyway - good to move it.